File: app/src/main/java/com/lazylee/lzywanandroid/activity/enter/EnterActivity.java
package com.lazylee.lzywanandroid.activity.enter;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.lazylee.lzywanandroid.R;
import com.lazylee.lzywanandroid.activity.login.LoginActivity;
import com.lazylee.lzywanandroid.activity.main.MainActivity;

public class EnterActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.enter_activity);
    }

    @Override
    protected void onStart() {
        super.onStart();
        startActivity(new Intent(this, MainActivity.class));
    }
}

File: app/src/main/java/com/lazylee/lzywanandroid/ui/activity/enter/EnterActivity.kt
package com.lazylee.lzywanandroid.ui.activity.enter

import android.content.Intent

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import com.lazylee.lzywanandroid.R
import com.lazylee.lzywanandroid.ui.activity.main.MainActivity

class EnterActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.enter_activity)
    }

    override fun onStart() {
        super.onStart()
        //startActivity(new Intent(this, MainActivity.class));
    }
}
